eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -175,3 +179,12 @@ def _force_create_enrollment(username, course_id, mode, is_active):
     except Exception as err:  # pylint: disable=broad-except
         raise APIException(repr(err))
     return enrollment
+
+
+def _validate_org(course_id):
+    """
+    Validates the course organization against all possible orgs for the site
+    """
+    course_key = CourseKey.from_string(course_id)
+
+    return course_key.org in get_current_site_orgs()
```

Comment:
Is this logic getting an update soon @diegomillan ?

Reply:
Yes, Im on it @felipemontoya

Reply:
Ready to review here @felipemontoya